implementations/validator/validator.py

```diff
@@ -113,7 +136,8 @@ def do_test(self):
         return self.format_response(infojson, fmt)
 
     def dispatch_views(self):
-        self.app.route("/validate", "GET", self.do_test)
+        self.app.route("/validate", "GET", self.do_GET_test)
+        self.app.route("/postvalidate", "POST", self.do_POST_test)
```

Comment:
I didn't want to lose the GET functionality for those that already had it in place. Shall I make a commit to POST to /validate?